services/data/src/links/RestAPILink/queryToRequestOptions/requestContentType.ts

```diff
@@ -25,20 +27,33 @@ const resourceExpectsMultipartFormData = (
         multipartFormDataMatcher(type, query)
     )
 
-export const FORM_DATA_ERROR_MSG =
-    'Could not convert data to FormData: object does not have own enumerable string-keyed properties'
+const resourceExpectsXWwwFormUrlencoded = (
+    type: FetchType,
+    query: ResolvedResourceQuery
+) =>
+    Object.values(xWwwFormUrlencodedMatchers).some(xWwwFormUrlencodedMatcher =>
+        xWwwFormUrlencodedMatcher(type, query)
+    )
+
+export const getConversionErrorMessage = (outputType: string): string =>
+    `Could not convert data to ${outputType}: object does not have own enumerable string-keyed properties`
```

Comment:
I think I would just do this inline on line 49. I didn't see this export used anywhere (or I missed it) and it simplifies the code.